addons/mail/static/tests/core/message_model_tests.js

```diff
@@ -37,7 +35,7 @@ QUnit.test("Message model properties", async (assert) => {
     assert.ok(message.isNeedaction);
     assert.strictEqual(message.body, "<p>Test</p>");
     assert.strictEqual(
-        moment(message.date).utc().format("YYYY-MM-DD hh:mm:ss"),
+        message.date.toFormat("yyyy-MM-dd hh:mm:ss"),
```

Comment:
serializeDateTime here?

Reply:
note sure (maybe ask BOI)

Reply:
if not HH:mm:ss seems to be what is appropriate (see dates.js)

Reply:
yes true